client/my-sites/media/main.jsx

```diff
@@ -336,4 +358,8 @@ class Media extends Component {
 	}
 }
 
-export default localize( Media );
+const mapStateToProps = state => ( {
+	selectedSite: getSelectedSite( state ),
```

Comment:
Doesn't hurt to also request `selectedSiteId` directly here:
```jsx
selectedSiteId: getSelectedSiteId( state ),
```

Reply:
Though, there is also a `selectedSite.slug` used, so, while it makes sense to directly select both, it'd also maybe clutter `mapStateToProps` too much?

Reply:
I avoided doing this because `selectedSite` object is passed to `EditorMediaModalDetail` and `MediaLibrary` components. I'd rather leave that refactor for a separate PR.